docs/source/user/jitclass.rst

```diff
@@ -10,7 +10,7 @@ Compiling Python classes with ``@jitclass``
   exposed or implemented, yet.
 
 
-Numba supports code generation for classes via the :func:`numba.jitclass`
+Numba supports code generation for classes via the :func:`numba.experimental.jitclass`
```

Comment:
Please line wrap to 80 characters, thanks!

Reply:
Happy to do so; it will just increase the diff size.

Reply:
Thank you :)